packages/orchestrator-ui-components/src/types/subscription.ts

```diff
@@ -1,43 +0,0 @@
-import { ProductDefinition } from './types';
```

Comment:
I think it is better to split the types.ts into different "subjects", just like this file. The types file is getting quite big.

Reply:
I was getting circular dependecies by having these types in a separate file. That's why I put them together